### DISCUSSION TITLE
use video tag instead of canvas

### DISCUSSION THREAD

**Discussion by waleedd32** [k-yle/rtsp-relay#146] [Q&A]
Hi, any idea how can i replace  < canvas ref={ canvas } /> with < vide></ video> ?  

import React, { useRef, useEffect } from 'react';
import ReactDOM from 'react-dom';
import { loadPlayer } from 'rtsp-relay/browser';

function App() {
  const canvas = useRef(null);

  useEffect(() => {
    if (!canvas.current) throw new Error('Ref is null');

    loadPlayer({
      url: 'ws://.../api/stream',
      canvas: canvas.current,
    });
  }, []);

  return (
    < div>
      {' '}
      < div>
        < canvas ref ={ canvas} />
      < / div>
      < video>< / video >
    < / div>
  );
}

export default  App;

**Comment by k-yle** [comment]
You can't, jsmpeg draws onto a canvas. There is no videoStream.
